### PR TITLE
test: async/await

### PR DESCRIPTION
**Pull request by bcollamore**

### CODE REVIEW

File: Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs [hunk 1]
```diff
@@ -9,7 +10,7 @@ public abstract class AssertDiagnosticVerifier : DiagnosticVerifier
 	{
 		private readonly AssertCodeHelper _helper = new();
 
-		protected void VerifyError(string methodBody, string expectedDiagnosticId)
+		protected async Task VerifyErrorAsync(string methodBody, string expectedDiagnosticId)
```
Comment (ynsehoornenborg) [inline Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs:13]: This method has the Async postfix, please remove it

File: Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs [hunk 2]
```diff
@@ -20,14 +21,14 @@ protected void VerifyError(string methodBody, string expectedDiagnosticId)
 					Severity = DiagnosticSeverity.Error,
 					Location = new DiagnosticResultLocation("Test0.cs", null, null),
 				};
-			VerifyDiagnostic(test, expected);
+			await VerifyDiagnostic(test, expected).ConfigureAwait(false);
 		}
 
-		protected void VerifyNoError(string methodBody)
+		protected async Task VerifyNoErrorAsync(string methodBody)
```
Comment (ynsehoornenborg) [inline Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs:27]: This method has the Async postfix, please remove it

File: Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs [hunk 3]
```diff
@@ -60,19 +60,19 @@ protected void VerifyError(string methodBody, string methodAttributes, int expec
 				expected.Message = new Regex(error, RegexOptions.Singleline, TimeSpan.FromSeconds(1));
 			}
 
-			VerifyDiagnostic(test, expected);
+			await VerifyDiagnostic(test, expected).ConfigureAwait(false);
 		}
 
-		protected void VerifyError(string methodBody, int expectedErrorLineOffset = 0, int expectedErrorColumnOffset = 0, string error = null)
+		protected async Task VerifyErrorAsync(string methodBody, int expectedErrorLineOffset = 0, int expectedErrorColumnOffset = 0, string error = null)
```
Comment (ynsehoornenborg) [inline Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs:66]: This method has the Async postfix, please remove it

**Review by ynsehoornenborg — CHANGES_REQUESTED** [review]
The `Async` postfix on the base classes breaks overloading, please fix those at least

File: Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs (hunk 1 above)
Comment (bcollamore) [reply Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs:13]: done

File: Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs (hunk 2 above)
Comment (bcollamore) [reply Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs:27]: done

File: Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs (hunk 3 above)
Comment (bcollamore) [reply Philips.CodeAnalysis.Test/Verifiers/AssertCodeFixVerifier.cs:66]: done

**Review by ynsehoornenborg — APPROVED** [review]

**Pull request merged**